places

Dryksd opened the pull request.

Dryksd committed:
выбран один шрифт Roboto-Black.ttf и подключен

Dryksd committed:
выбран один шрифт Roboto-Black.ttf и подключен
добавлена директория assets для ресурсов

Jokerkras commented on pubspec.yaml:
```diff
@@ -24,7 +24,7 @@ flutter:
 
   assets:
-    - res/Roboto/
+    - assets/res/Roboto/Roboto-Black.ttf
```
https://flutter.dev/docs/cookbook/design/fonts

Dryksd committed:
подключен шрифт Roboto-Black.ttf (иным способом)

Dryksd committed:
добавил два виджета StateLess и StateFull - MyFirstWidjet и MySecondWidjet

Dryksd committed:
добавил виджет App. Добавил функцию getContextType() в два виджета. MyFirstWidget и MySecondWidget. Добавил поле title в виджет App.

Dryksd committed:
Создал директорию ui,screen и файл sight_list_screen.dart. В нем создал класс SightListScreen (statefull) и подключил его к home в main.dart (в целях отладки).

Dryksd committed:
Создал директорию ui,screen и файл sight_list_screen.dart. В нем создал
класс SightListScreen (statefull) и подключил его к home в main.dart
(в целях отладки).в прошлой ветке забыл коммит сделать.

Dryksd committed:
Сделал Appbar в sight_list_screen.dart с Text

Dryksd committed:
Сделал Appbar в sight_list_screen.dart с RichText

Dryksd committed:
Сделал Appbar в sight_list_screen.dart с RichText и увеличил шрифт и его же сделал пожирнее. Отформатировал код.

Dryksd committed:
увеличил шрифт, сделал пожирнее, убрал new

Dryksd committed:
создал в папке lib папку domain.

В папке domain создал файл sight.dart. Файл содержит модель данных Sight с полями

В папке lib создал файл mocks.dart. Файл содержит коллекцию моковых достопримечательностей final List mocks = [];

Инициализировал несколько интересных мест

Dryksd committed:
переименовал sight_card.dart в sight.dart

Dryksd committed:
добавил несколько файлов

Dryksd committed:
добавил файл Color,убрал лишние комменты, добавил описание к классам

Dryksd committed:
выполнил задания.номера заданий пометил в комментариях.

Dryksd committed:
фон заменил картинками и добавил прогресс индикатор к каждой картинке.
 В комментариях указал где картинки и прогресс индикатор.

Dryksd committed:
создал экран VisitingScreen

Dryksd committed:
создал в main.dart выбор темной и светлой темы.

Dryksd committed:
Создал экран FilterScreen.

Dryksd committed:
Добавил InkWell в FilterScreen.Там же изменил значение в 32 строке у _value.

Dryksd committed:
убрал Container в sight_card, потому что он там не нужен.

Dryksd committed:
5.4

Dryksd committed:
5.4

Dryksd committed:
5.4 изменения внешнего вида.изменение функции в SightSearchScreen

Dryksd committed:
5.6 реализовано удаление карточки посещенных(непосещенных)мест. Применен глобал кей. Для эксперимента и задания.
